fastlane/lanes/release.rb

```diff
@@ -364,4 +364,8 @@ def bundle_file_path(app, version)
     prefix = APP_SPECIFIC_VALUES[app.to_sym][:bundle_name_prefix]
     File.join(ENV['PROJECT_ROOT_FOLDER'], 'build', "#{prefix}-#{version['name']}.aab")
   end
+
+  def signed_apk_path(app, version)
+    bundle_file_path(app, version).sub!('.aab', '.apk')
```

Comment:
If I'm reading this correctly, the call site in your `download_signed_apk_from_google_play` lane calls this with `version` being a `Hash`, not à `String` like this code seems to expect?

Also, I'm surprised using `sub!`, aka the mutating-in-place version of `sub` (that's what the `!` in the method name means), on the return value of a function call does not crash. I'm pretty sure in strict mode it would? And there's no real need to mutate in place here anyway?

Reply:
Addressed in 5727395955adc4cf38ebe77c4a217c727eea0370.